stream-chat-android-ui-components/src/main/res/layout/stream_ui_message_reply_view.xml

```diff
@@ -45,47 +45,30 @@
         >
 
         <FrameLayout
-            android:id="@+id/logoContainer"
+            android:id="@+id/attachmentContainer"
```

Comment:
Okay this explains the logoContainer change

Reply:
Yup, it becomes just the root for the attachment.